tests/foreman/cli/test_satellitesync.py

```diff
@@ -619,31 +631,46 @@ def test_positive_export_format_inheritance(
         export_1['id'] = history[-1]['id']
 
         # Run an incremental export with no format provided.
-        export_2 = inc_export(exp_params)
+        inc_export(exp_params)
 
         # Assert that the format of the export is "importable" (not "syncable").
-        assert importable_msg in export_2['message']
```

Comment:
This was meant to check the correct path is printed in stdout by the export. But we can omit if we are sure the same gets into history.